src/components/multiStepBar/MultiStepBar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {IClassName} from '../../utils/ClassNameUtils';
+import {ITooltipProps, Tooltip} from '../tooltip/Tooltip';
+
+export type IMultiStepState = 'to-do' | 'doing' | 'done' | 'error' | 'warning';
+
+export interface IStep {
+    state: IMultiStepState;
+    text?: string;
+    tooltip?: ITooltipProps;
+}
+
+export interface IMultiStepBarProps {
+    className?: IClassName;
+    small?: boolean;
+    separator?: boolean;
+    inProgressAnimation?: 'slide' | 'shine';
+    flareAnimation?: boolean;
```

Comment:
To keep the same logic as inProgressAnimation we should probably use `animation?: '' | 'flare';`